tasks/utilities/task_czgenepi_wrangling.wdl

```diff
@@ -18,26 +18,57 @@ task czgenepi_wrangling {
     String county_column_name
 
     # optional inputs
-    String gisaid_id_column_name
     String genbank_accession_column_name
     String sequencing_date_column_name
-    String sample_is_private_column_name
+
+    # setting columns
+    String organism
+    Boolean is_private
 
     # runtime
     Int disk_size = 100
   }
   command <<<
+    export is_private=~{is_private}
+   
     # parse terra table for data
     python3 <<CODE
     import pandas as pd
     import numpy as np
     import re
     import os
 
+    # set a function to grab only the year from the date
+    def year_getter(date):
+      r = re.compile('^\d{4}-\d{2}-\d{2}')      
+      if pd.isna(date):
+        print("Incorrect collection date format; collection date must be in YYYY-MM-DD format. Invalid date was: NaN")
+      elif r.match(date) is None:
+        print("Incorrect collection date format; collection date must be in YYYY-MM-DD format. Invalid date was: " + str(date))
+        return np.nan
+      else:
+        return date.split("-")[0]
+
+    if os.environ["is_private"] == "true":
```

Comment:
Will python error out if `is_private` isn't set? May be worth setting it default to false